src/app/commons/header/header.component.ts

```diff
@@ -6,5 +6,13 @@ import { Component } from '@angular/core';
   styleUrls: ['./header.component.css']
 })
 export class HeaderComponent {
+  isToggle: boolean = false;
+  constructor() { }
+  toggleMenu() {
+    this.isToggle = !this.isToggle;
+  }
 
+  onVisited() {
+    this.isToggle = !this.isToggle;
```

Comment:
Both the functions are doing the same logic. You can collapse these down to one function.